src/status_im/subs/wallet/wallet.cljs

```diff
@@ -48,11 +48,10 @@
  :wallet/home-tokens-loading?
  :<- [:wallet/tokens-loading]
  (fn [tokens-loading]
-   (or (empty? tokens-loading)
```

Comment:
definitely safe?

Reply:
@J-Son89 only definitely in the sense that I tested, and that QAs will test :)